packages/main/src/plugin/kubernetes/resource-informer.ts

```diff
@@ -38,11 +44,20 @@ export interface OfflineEvent extends BaseEvent {
   reason?: string;
 }
 
+export interface ResourceInformerOptions<T extends KubernetesObject> {
+  kubeconfig: KubeConfigSingleContext;
+  path: string;
+  listFn: ListPromise<T>;
+  kind: string;
+  plural: string;
+}
+
 export class ResourceInformer<T extends KubernetesObject> implements Disposable {
   #kubeConfig: KubeConfigSingleContext;
   #path: string;
   #listFn: ListPromise<T>;
-  #resourceName: string;
+  #pluralName: string;
+  #kindName: string;
```

Comment:
Could you add a short comment on each of them to precise their prupose, I never know which one is what :crab: